src/api/party.ts

```diff
@@ -39,7 +49,42 @@ export const patchPartyDetail = async (
 ) => {
   try {
     const response = await axiosInstance.patch(`/parties/${partyId}`, partyAPIBody);
-    return response.data;
+    if (response) return response.data;
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+// 멤버 방출
+export const deleteMemberBanish = async ({
+  partyId,
+  memberId,
+}: deleteMemberBanishProps) => {
+  try {
+    const response = await axiosInstance.delete(
+      `/parties/${partyId}/members/${memberId}`
+    );
+    if (response) return response.data;
```

Comment:
함수들 깔끔하게 잘 정리했는걸👍
근데 delete 같은 것들은 response가 없을텐데 굳이 써줘야하나 싶어
```suggestion
    await axiosInstance.delete(
      `/parties/${partyId}/members/${memberId}`
    );
```

Reply:
앗 그르넹ㅋㅋㅋ 수정하겠습니당